pkg/archive/archive_test.go

```diff
@@ -214,14 +213,14 @@ func testArchive(t *testing.T, when spec.G, it spec.S) {
 			})
 		})
 
-		when("includeRoot is true", func() {
-			it("sets metadata on base dest file", func() {
+		when("mode is set to 0755", func() {
```

Comment:
Added test, right?

Reply:
Yeah, We had just the test for the 0777 but since we were defaulting to it when `includeRoot` I also wanted to prove/document through tests that we are also capable to set the mod to `0755` if `includeRoot` is false and we pass a different mode than `0777` and `-1`